src/containers/org/DataSetMetaContainer.js

```diff
@@ -29,12 +44,34 @@ const DataSetMetaContainer = ({
 
   const [tableData, setTableData] = useState([]);
 
+  const metadata :Map = useSelector((store) => store.getIn([SHIPROOM, 'metadata']));
+  const metadataRS :?RequestState = useRequestState([SHIPROOM, GET_SHIPROOM_METADATA]);
   const propertyTypes :Map<UUID, PropertyType> = useSelector(selectEntitySetPropertyTypes(dataSetId));
   // BUG: unfortunately, I think hashCode() is computed every time since the Map is always "new"
   const propertyTypesHash :number = propertyTypes.hashCode();
 
+  const parsedColumnInfo = useMemo(() => {
+    const columninfo :List = getPropertyValue(metadata, FQNS.OL_COLUMN_INFO, List());
+    try {
+      const parsedColumns = JSON.parse(columninfo.first());
+      const parsedColumnInfoWithIds = parsedColumns.map((column, index) => ({
+        id: column.propertyTypeId,
```

Comment:
This will be `undefined` for atlas data sets.

Reply:
that's ok. the object spread occurs afterwards. this is really only used for react key for repeated elements